installer/installer.go

```diff
@@ -36,10 +37,10 @@ func NewInstaller(l log.Logger) *Installer {
 		logger:        l,
 	}
 	if err := installer.openDB(); err != nil {
-		panic(err)
+		shutdown.Fatalf("Error opening database: %s", err)
```

Comment:
This should explicitly check for the "another instance is running" error and phrase it as such.